### ISSUE TITLE
RTD: fix edit on github links

### ISSUE THREAD

**Issue by hinerm** [scijava/scijava#277]
Prior to https://github.com/scijava/scijava/commit/907e10b1f1dcf2251fb795e03fbe95a19e1dd1f4 they were going to a root of `github.com/scijava/scijava/blob/main/docs` but missing the `/ops/doc` sub-directories.

**Comment by gselzer** [comment]
Additionally, now I see them saying "View page source" whereas they previously said "Edit on Github"...

**Comment by ctrueden** [comment]
@gselzer Yes, it changed to "View page source" after I pushed 907e10b1f1dcf2251fb795e03fbe95a19e1dd1f4

**Comment by gselzer** [comment]
Sure. Ideally I think that the Github link would be nice. Do you know if it's possible to retain that?

**Comment by hinerm** [comment]
sorry for any confusion, the goal of this issue was to go back to a link to the source on GitHub, but with that link fixed of course.

**Comment by ctrueden** [comment]
> Do you know if it's possible to retain that?

No, I have no idea. I did some web searches and found the `github_url` property and tried setting it, that's all. But it doesn't work the way I hoped it would.

**Comment by ctrueden** [comment]
@hinerm How does your commit fix the issue? It merely resets things to the broken "Edit on GitHub" links with incorrect paths, no?

**Comment by gselzer** [comment]
It's actually this commit that does the lifting: 4848b3d
